docs/dashboards/interactive.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Interactive dashboards
+title: Interactivity on dashboards
```

Comment:
This one has been bothering me for a LONG time. It's so weird to have "interactive dashboards" as our main value prop, but then have a docs article on this? Because it feels like, all our dashboards are interactive, so this must be just a generic article about dashboards? but it's not

On the other hand, I can never find the article on e.g. custom destinations (this is the article)

I don't think this is the best name, but I think it's better than the current one